### PR TITLE
Add filter on itemReviewed

### PR DESCRIPTION
**Pull request by soyuka**
todo: add test

### CODE REVIEW

File: app/config/services.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ parameters:
 #    parameter_name: value
 
 services:
+    review.search_filter:
+        parent: 'api_platform.doctrine.orm.search_filter'
+        arguments: [ { itemReviewed: 'exact' } ]
+        tags: [ { name: 'api_platform.filter' } ]
```
Comment (soyuka) [inline app/config/services.yml]: Do we still need an `id` here?

**Comment by soyuka** [comment]
I've fixed behat tests on the demo :). You may need to drop/create the tables in the demo because some fileds changed. Maybe `schema:update` will do. @dunglas  did you create books fixtures or this is just users that tested the demo?

**Comment by soyuka** [comment]
May we also change CORS to allow more demos (now CORS are only allowed on "http://localhost")?

**Comment by dunglas** [comment]
Yes, we should set it to `*` for the demo.

**Pull request merged**